### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by dylan-gonzalez** [Monash-Connected-Autonomous-Vehicle/autoware#20] [Announcements]
Hey @Monash-Connected-Autonomous-Vehicle/avp-team 

Thanks for coming to the meeting today (sorry for the technical difficulties)

### Please do the following (and react when you have):

- Fill out the [AVP M1 Task Selection](https://forms.gle/JbWCaiHVRS1KE23J9) google form
- Request to join the [Github Team](https://github.com/orgs/Monash-Connected-Autonomous-Vehicle/teams/avp-team)
- Read through the [Software Guidelines](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/wiki/Software-Guidelines) on the wiki. _especially the part about MCAV PC Usage_
- Take a look at the [AVP Diagram](https://app.diagrams.net/#G1T9vIuZJyE1f5AhWn4aL91sWi9EbfGOO_). Go to View->Tags to filter out different milestones (only show one) - ask questions here if you have any doubts about the diagram
- Download the Github Mobile app and configure your notifications so you don't miss any thing
- Fill out the [Days in the workshop notion](https://www.notion.so/monashcav/Days-in-the-workshop-beac6f0f115c4ba09463075b9a020eb1) for this week

**General Info**

- Slides from today can be found [here](https://docs.google.com/presentation/d/1blU79Ptixa6uPDCLfO2Xgyq_KwQOQWQpCGCLhjdYAQk/edit#slide=id.p)
- We will be using Github Milestones to attach to each relevant issue - we have made some [here](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/milestones).
- We will be updating the AVP Diagram this week to include the later milestones (e.g. the infrastructure stuff)
- Milestones and Wiki is attached to the [autoware ](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware)repo. Discussions can be found at the organization level.

**Comment by dylan-gonzalez** [comment]
@Monash-Connected-Autonomous-Vehicle/avp-team 

**Everyone should have been assigned to a Github Issue by now**

You should all be working with at least one other person - schedule some working times to work with your team members, fill out the [Working days in Workshop notion](https://www.notion.so/monashcav/Days-in-the-workshop-beac6f0f115c4ba09463075b9a020eb1). Remember that you are required to be in the workshop _at least_ 1 day a week.

There are also deadlines assign to each of these issues - please try to stick these as much as possible. If you need help, reach out, make a discussion, etc.

We will also have our weekly meetings Monday 12-2pm in Workshop 1 to go over each of these issues and how everyone is tracking - it's easier to do these in person so if you can please attend in person, otherwise zoom will be available too.

And please join the [Github Team](https://github.com/orgs/Monash-Connected-Autonomous-Vehicle/teams/avp-team)!!!

**Reply by AbBaSaMo** [reply]
Hi
